src/index.js

```diff
@@ -75,6 +75,7 @@ export default function TimeAgo({
     if (!live) {
       return
     }
+    setTimeNow(now());
```

Comment:
`timeNow` is changed from `useEffect` hook that has it as a dependency

Reply:
Oh you're right. And more observant than my IDE's linter :-)

It does work though. I guess that's because my machine/browser is fast enough to re-execute before the time actually changes at the millisecond resolution that is used here. Not great.

I can't think of an elegant solution, so I'll update to the least ugly one I can think of.